web/app/controllers/bloggs_controller.rb

```diff
@@ -12,17 +12,19 @@ def show
 
   def new
     @article = Blogg.new
+    authorize! :new, @article
```

Comment:
Adding authorize! calls to ensure the user is an admin before they can create a new blogg